SG14_test/slot_map_test.cpp

```diff
@@ -86,6 +86,24 @@ struct Vector {
         a = std::move(b);
         b = std::move(t);
     }
+    iterator erase(const_iterator first, const_iterator last) {
```

Comment:
Why is this member function required? (If it's really required, add a unit test that fails before this addition and succeeds afterward.)

You should also update [the exposition-only SlotMapContainer concept](https://github.com/WG21-SG14/SG14/blob/f53bc87714cc3f18049a98d15ddae8cdadec867a/SG14_test/slot_map_test.cpp#L541) if `erase` is required.

Reply:
It is to convert a `const_iterator` to `iterator`. I'll add it to the concept test.